N.csv

```diff
@@ -0,0 +1,5 @@
+Lemma,Sense,Word Class,Year,Quotation,Source,Note
+奶糖,/, noun, 1886, 中國資料 洪士提反譯《萬國藥方》卷一
```

Comment:
```suggestion
奶糖,/,noun,1886,,洪士提反譯『萬國藥方』卷一,
```
Follow this for other entries and add the quotations. Hover over Schmidt’s rows to reveal the quotations. (Refer to first email today.) If he doesn’t provide one, show it like this `,,` because you still need a blank column there.